docs/pages/versions/unversioned/sdk/webbrowser.mdx

```diff
@@ -59,7 +59,7 @@ const styles = StyleSheet.create({
 
 ### Handling deep links from the WebBrowser
 
-If you use the `WebBrowser` window for authentication or another use case where you want to pass information back into your app through a deep link, add a handler with `Linking.addEventListener` before opening the browser. When the listener fires, you should call [`dismissBrowser`](#webbrowserdismissbrowser). It will not automatically be dismissed when a deep link is handled. Aside from that, redirects from `WebBrowser` work the same as other deep links. Read more about it in [Linking](/guides/linking#handling-links).
+If you use the `WebBrowser` window for authentication or another use case where you want to pass information back into your app through a deep link, add a handler with `Linking.addEventListener` before opening the browser. When the listener fires, you should call [`dismissBrowser`](#webbrowserdismissbrowser). It will not automatically be dismissed when a deep link is handled. Aside from that, redirects from `WebBrowser` work the same as other deep links. Read more about it in [Linking](/linking/into-your-app/#handle-urls).
```

Comment:
this could be a follow-up for later, since you didn't really change this, but this advise is pre-Expo-Router.

Reply:
Do we still keep this info as part of non Expo Router projects and say Expo Router handles deep links automatically?

Reply:
To resolve it, I've added the following here (backported to other API references - 51 and 50):

![CleanShot 2024-10-02 at 18 40 49@2x](https://github.com/user-attachments/assets/91147bf0-5ffa-4d2f-af3e-2b812691e2e0)

![CleanShot 2024-10-02 at 18 40 42@2x](https://github.com/user-attachments/assets/198a3a1e-0d32-435d-b97e-81265a857d30)

Let me know what you think.